Marlin/Configuration.h

```diff
@@ -143,7 +143,7 @@
 #if ENABLED(SWITCHING_EXTRUDER)
   #define SWITCHING_EXTRUDER_SERVO_NR 0
   #define SWITCHING_EXTRUDER_SERVO_ANGLES { 0, 90 } // Angles for E0, E1[, E2, E3]
-  #if (EXTRUDERS == 4)
+  #if EXTRUDERS >= 3
     #define SWITCHING_EXTRUDER_E23_SERVO_NR 1
   #endif
```

Comment:
We can make it so the 3rd extruder just isn't switching. That will allow for a Dual X setup with one dual-extruder, plus one single extruder.